jest.base.config.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  globals: {
+    'ts-jest': {
```

Comment:
We have configuration in here for `jest-preset-angular` but that library is not installed. Do our tests currently run? Or is that going to be a follow up ticket? (Just want to make sure the ticket is created if so)

Reply:
Yes, the tests run, we don't have jest-preset-angular. I don't know if it's needed to be installed, or not.

I think we'd benefit from a 'setup jest' ticket once we flesh out what's needed. Jest is not the focus of this ticket, but I got to a place where I needed to get the tests to run.